docs/user-guide/remapping.ipynb

```diff
@@ -0,0 +1,1325 @@
+{
```

Comment:
<p>I'm going to make a very basic <code>quad-triangle</code> grid that you can use as the destination here, since remapping to the same grid could be a little confusing to the reader. </p><p> </p><p>This example is great though! Very minimal and get the point across very nicely. </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UXARRAY/uxarray/pull/807/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>